src/MUIDataTable.js

```diff
@@ -332,6 +332,8 @@ class MUIDataTable extends React.Component {
     let filterData = [];
     let filterList = [];
 
+    let sortEnabled = false;
```

Comment:
Since we have a `sort` option as well as a `sortDirection` option, the var name might be a little more clear if it were something like `sortDirectionSet` instead.